packages/url_launcher/url_launcher_web/test/build.yaml

```diff
@@ -0,0 +1,6 @@
+targets:
+  $default:
+    sources:
+      - integration_test/*.dart
```

Comment:
This file is required so the build process is able to find the `@GenerateMocks` annotation